docs/examples/talk-to-a-video.md

```diff
@@ -0,0 +1,83 @@
+We can use Google Gemini's [native video input](https://ai.google.dev/gemini-api/docs/vision?lang=python#prompting-video) capabilities to ask questions about [a video](https://www.youtube.com/watch?v=XXuIBHO4qa8).
+In this example, we upload a video file using Gemini's file API, and then pass the result using the [GenericArtifact](../reference/griptape/artifacts/generic_artifact.md) to the Agent.
+Note that because we are using Gemini-specific features, this will not work with other [Prompt Drivers](../griptape-framework/drivers/prompt-drivers.md).
+
+```python
+import time
+from griptape.structures import Agent
+from griptape.tasks import PromptTask
+from griptape.artifacts import ListArtifact, GenericArtifact, TextArtifact
+from griptape.config import GoogleStructureConfig
+import google.generativeai as genai
+
+video_file = genai.upload_file(path="tests/resources/griptape-comfyui.mp4")
```

Comment:
do we want to put a wrapper around this at all so that a user doesn't need to configure both Griptape and the genai API with a key?

Reply:
I'm not really sure what to wrap it in since they're the only provider that offers this.

We could put it in the Prompt Driver, but then the API diverges. This is a but clunky but it's kind of an interesting way to show how you can mix griptape with native SDK functionality.

Reply:
That's a good point on the mix of native SDK + Griptape. We can leave it at that for the V1.

One thing I am thinking about for the future is if we can use our `StructureConfig` as a generic config to make API calls? It has the entire config to make it work. So essentially like `GoogleStructureConfig().api_call('upload_file', args={'path':'tests/resources/griptape-comfyui.mp4'})` type of thing.

Anyhow, future things if it comes up again.